jekyll/_cci2/language-ruby.md

```diff
@@ -7,60 +7,58 @@ categories: [language-guides]
 order: 8
 ---
 
-*[Tutorials & 2.0 Demo Apps]({{ site.baseurl }}/2.0/tutorials/) > Language Guide: Ruby*
+*[Tutorials & 2.0 Sample Apps]({{ site.baseurl }}/2.0/tutorials/) > Language Guide: Ruby*
 
 This guide will help you get started with a Ruby on Rails application on CircleCI. 
 
 ## Overview
 
 If you’re in a rush, just copy the sample configuration below into a `.circleci/config.yml` in your project’s root directory and start building.
 
-Otherwise, we recommend reading our [walkthrough](#config-walkthrough) for a detailed explanation of our configuration.
-
-We maintain a reference Ruby on Rails project to show how to build Ruby on CircleCI 2.0:
+CircleCI maintains the sample Ruby on Rails project at the following links:
 
 - <a href="https://github.com/CircleCI-Public/circleci-demo-ruby-rails" target="_blank">Demo Ruby on Rails Project on GitHub</a>
 - <a href="https://circleci.com/gh/CircleCI-Public/circleci-demo-ruby-rails" target="_blank">Demo Ruby on Rails Project building on CircleCI</a>
 
 In the project you will find a commented CircleCI configuration file <a href="https://github.com/CircleCI-Public/circleci-demo-ruby-rails/blob/master/.circleci/config.yml" target="_blank">`.circleci/config.yml`</a>.
 
-We've built this application using the latest stable Rails version 5.1, `rspec-rails`, and [RspecJunitFormatter][rspec-junit-formatter] with PostgreSQL as our database of choice.
+The application uses the latest stable Rails version 5.1, `rspec-rails`, and [RspecJunitFormatter][rspec-junit-formatter] with PostgreSQL as the database.
 
-This application also assumes you're using one of our [CircleCI Images](http://circleci.com/docs/2.0/circleci-images).
+This application build also uses one of the pre-built [CircleCI Docker Images](http://circleci.com/docs/2.0/circleci-images).
```

Comment:
Is it an application or a build? I'm not sure what an "application build" is.

Reply:
What do you suggest as a change to make this more clear?

Reply:
I would choose one or the other. In this case, I think "build" probably makes more sense, although "application's configuration" might be more accurate.